<launch>
    <node pkg="camera_location_module" type="server_camera_location.py" name="camera_location_interpreter" required="true">
    </node>
    <node pkg="height_map_driver" type="height_map_wrapper.py" name="height_map">
    </node> 
    <node pkg="biomass_estimator_driver" type="biomass_estimator_wrapper.py" name="biomass_estimate">
    </node>
    <node pkg="biomass_map_driver" type="biomass_map.py" name="biomass_csv_server" required="true" output="screen">
    </node>
    <node pkg="biomass_map_driver" type="biomass_map_wrapper.py" name="biomass_map" output="screen">
    </node>
</launch>